.gitignore

```diff
@@ -51,3 +51,6 @@ flycheck-*
 
 ### vim
 .ropeproject/
+
+### VSCode
+.devcontainer
```

Comment:
Please don't add more. Add those to your local .gitignore (I'm going to remove the other ones soon).